pkg/performanceprofile/cmd/render/cmd.go

```diff
@@ -74,6 +77,11 @@ func (r *renderOpts) readFlagsFromEnv() {
 	if assetsOutDir := os.Getenv("ASSET_OUTPUT_DIR"); len(assetsOutDir) > 0 {
 		r.assetsOutDir = assetsOutDir
 	}
+	if addOwnerRef := os.Getenv("ADD_OWNER_REF"); len(addOwnerRef) > 0 {
```

Comment:
nit: you probably followed the old code but we should use `os.LookupEnv`

Reply:
I did. For the options we're using here, there's not much difference though, because empty string is not a legal value so we can use empty string to umambibuously mean "value not set"